content/doc/upgrade/from-gogs.en-us.md

```diff
@@ -61,8 +60,10 @@ ROOT_PATH = /home/:USER/gitea/log
 ```
 
 * Verify by starting Gitea with `gitea web`
-  
-  
+
+### Troubleshooting
+* If you encounter errors relating to custom templates in the `gitea//custom/templates` folder, try moving the templates causing the errors away one by one. They may not be compatible with Gitea.
```

Comment:
Fixed the double forward slash, but I don't understand "Missing new line before this"